less/mcq.less

```diff
@@ -132,8 +144,7 @@
         .mcq-item-icon {
             display:none;
         }
-        .incorrect .selected .mcq-correct-icon,
-        .correct .selected .mcq-correct-icon {
+        .selected .mcq-correct-icon {
```

Comment:
Is there a reason this selector has been modified? This will break the `_canShowMarking` functionality introduced in https://github.com/adaptlearning/adapt-contrib-mcq/pull/126.